content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -71,6 +71,7 @@ different Kubernetes components.
 | `DynamicVolumeProvisioning` | `true` | GA | 1.8 | |
 | `EnableEquivalenceClassCache` | `false` | Alpha | 1.8 | |
 | `ExpandInUsePersistentVolumes` | `false` | Alpha | 1.11 | |
```

Comment:
pls change line 73 to:

```
| `ExpandInUsePersistentVolumes` | `false` | Alpha | 1.11 | 1.13 |
```
If the alpha state was ended in 1.13.

Reply:
Done